packages/plugin-eslint/package.json

```
@@ -2,6 +2,13 @@
   "name": "@code-pushup/eslint-plugin",
   "version": "0.49.0",
   "license": "MIT",
+  "description": "[Code PushUp](https://github.com/code-pushup/cli/tree/main/packages/cli/README.md) plugin for detecting problems in source code using ESLint.📋",
```

```suggestion
  "description": "[Code PushUp](https://github.com/code-pushup/cli/tree/main/packages/cli/README.md) plugin for detecting problems in source code using ESLint 📋",
```
other descriptions also don't have the dot